templates/hello-world/hydrogen.generated.d.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable eslint-comments/no-unlimited-disable */
+/* eslint-disable */
+import * as HydrogenStorefront from '@shopify/hydrogen-react/storefront-api-types';
+
+export type LayoutQueryVariables = HydrogenStorefront.Exact<{
+  [key: string]: never;
```

Comment:
We don't really have much control over the generated operations since they come from `typescript-operations` plugin. The type for empty vars is [hardcoded here](https://github.com/dotansimha/graphql-code-generator/blob/7e365b96aead8ce5a85be2edbfc6ad23e9501848/packages/plugins/typescript/operations/src/visitor.ts#L135).

We could overwrite the whole file with regexps but I think it's not worth it, no? 🤔